data/data/aws/cluster/vpc/sg-master.tf

```diff
@@ -289,6 +289,28 @@ resource "aws_security_group_rule" "master_ingress_kube_controller_manager_from_
   to_port   = 10257
 }
 
+resource "aws_security_group_rule" "master_ingress_cluster_policy_controller" {
```

Comment:
The AWS UPI should be updated with this as well. `upi/aws/cloudformation/03_cluster_security.yaml`

Reply:
Thanks,  updated.